src/mail/mjml/order_validated.mjml

```diff
@@ -0,0 +1,42 @@
+<mjml>
+  <mj-include path="./partial/header.mjml" />
+  <mj-body mj-class="bg--blue-100">
+    <mj-raw>{% load i18n static %}</mj-raw>
+    <mj-wrapper css-class="wrapper" padding="20px 40px 40px 40px">
+    <mj-section>
+      <mj-column>
+        <mj-image src="{% static 'joanie/images/logo_fun.png' %}" width="200px" align="left" alt="Company logo" />
+      </mj-column>
+    </mj-section>
+      <mj-section mj-class="bg--blue-100" border-radius="6px 6px 0 0" padding="30px 50px 60px 50px">
+        <mj-column>
+          <mj-text padding="0">
+            <p>
+              {%trans "Hello"%}{%if username%} {{username}} {%endif%}<br/>
+              <strong>{%trans "Thank you very much for your purchase!"%}</strong>
+            </p>
+          </mj-text>
+        </mj-column>
+      </mj-section>
+      <mj-section mj-class="bg--blue-100" padding="0 50px 60px 50px">
+        <mj-column>
+          <mj-text padding="0 0 20px 0">
+            <h1>{%trans "Your order has been confirmed."%}</h1>
+          </mj-text>
+          <mj-text padding="0">
+            {%trans "You have bought "%} <strong>{{product.title}}</strong>
+            <br />{%trans "for a price of"%} <strong>{{product.price}}</strong>.
```

Comment:
This would not be clear for translators and can be a problem with the order of words in some languages:
```suggestion
            {% blocktrans with title=product.title %}You have bought <strong>{{ title }}</strong>{% endblocktrans %}
            <br />{% blocktrans with price=product.price %}for a price of <strong>{{ price }}</strong>.{% endblocktrans %}
```

Reply:
I finally transform those two sentences if single one `blocktranslate`.